### PR TITLE
Added load balancer template and auxiliary scripts

### PR DESCRIPTION
**Pull request by abel-navarro**
For load balancer testing (LBaaS2) we are providing a heat template
to create two VMs to be the backends for the load balancer.

Since the Mitaka release does not support heat templates with LBaaS2,
auxiliary scripts have been created to provide this functionality just
for testing.

### CODE REVIEW

**Comment by abel-navarro** [comment]
[S57P10 (8) LBaaS migrations during non-disruptive upgrade](https://trello.com/c/usG7xO7K/364-s57p10-8-lbaas-migrations-during-non-disruptive-upgrade)

File: scripts/heat/load-balancer-create.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+# lbaasv2
+# https://docs.openstack.org/ocata/networking-guide/config-lbaas.html
+
+# Cirros image with thttpd and tcpdump enabled
+if [ ! -f cirros-thttpd.img ]; then
+  wget http://clouddata.nue.suse.com/images/openstack/x86_64/cirros-thttpd.img
+fi
+
+openstack image show cirros-thttpd
+if [ $? -ne 0 ]; then
```
Comment (jsuchome) [inline scripts/heat/load-balancer-create.sh]: Can't you just do `if ! openstack image show cirros-thttpd` ?

File: scripts/heat/load-balancer-create.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  openstack image create --disk-format qcow2 --container-format bare --public --file cirros-thttpd.img cirros-thttpd
+fi
+
+heat stack-show lbaas
+if [ $? -ne 0 ]; then
```
Comment (jsuchome) [inline scripts/heat/load-balancer-create.sh]: Same as above

File: scripts/heat/load-balancer-create.sh (hunk 1 above)
Comment (abel-navarro) [inline scripts/heat/load-balancer-create.sh]: that looks cleaner indeed. Updating.

File: scripts/heat/load-balancer-create.sh (hunk 2 above)
Comment (abel-navarro) [inline scripts/heat/load-balancer-create.sh]: same as above :)

**Comment by abel-navarro** [comment]
@jsuchome for now this script is executed manually to test the load balancers during upgrade. It wasn't part of the card to automate the testing, but this is providing enough help to do it.

**Pull request merged**